### ISSUE TITLE
[Accessibility- information structuring C9.3] Use of lists in the left sidebar

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#2998]
**Environment:** https://whitepaper-ft.meeds.io/

**Step of reproduction:**

Access to the left menu navigation

**Current behavior:**

All elements (Pages and spaces) are not structured by lists or created with invalid ARIA roles

**Expected behavior:**

All left menu items should be structured into lists to allow quick consultation of content => Use **ul/li** lists to structure sequences of elements of the same nature, as well as for successive links

![Image](https://github.com/user-attachments/assets/af4be334-953a-404e-a189-def46324b639)

**Comment by mbensalem** [comment]
still relevant and need decision
